packages/core/src/runtime/resolveStyleRules.ts

```diff
@@ -54,6 +55,14 @@ export function resolveStyleRules(
 ): [CSSClassesMap, CSSRulesByBucket] {
   // eslint-disable-next-line guard-for-in
   for (const property in styles) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (UNSUPPORTED_CSS_PROPERTIES.hasOwnProperty(property)) {
+      if (process.env.NODE_ENV !== 'production') {
+        console.error('Unsupported CSS property', property, 'will not be resolved');
```

Comment:
Let's make this error more human 🐱 

![image](https://user-images.githubusercontent.com/14183168/155694005-010d16a8-0163-453b-a960-23bd0c134637.png)

```suggestion
        console.error(
          [
            `@griffel/react: You are using unsupported shorthand CSS property "${property}". ` +
              `Please check your "makeStyles" calls, there *should not* be following:`,
            ' '.repeat(2) + `makeStyles({`,
            ' '.repeat(4) + `[slot]: { ${property}: "${styles[property as keyof GriffelStyle]}" }`,
            ' '.repeat(2) + `})`,
            '',
            'Learn why CSS shorthands are not supported: https://aka.ms/griffel-css-shorthands',
          ].join('\n'),
        );
```

Reply:
it's always easy to claim short error messages are easy to understand when you are writing them 😄